components/monitoring/controllers/base/kube-prometheus-stack.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: prometheus-community
+spec:
+  interval: 12h
+  type: oci
+  url: oci://ghcr.io/prometheus-community/charts
```

Comment:
This should be OCIRepository, see the cert-manager example please, also for metrics-server

Reply:
Fixed, thanks!